containers/versions.yml

```diff
@@ -3,7 +3,7 @@ docker_version:               "17.05.0~ce-0~debian-stretch"
 geth_version:                 "v1.8.12"
 golem_version:                "0.18.2"
 # Remember set version in concent_version variable also require configure it in the var-concent-<cluster_name>.yml file
-concent_version:              "v0.10.2-9-gccfae9af"
+concent_version:              "v0.10.2-71-gbcb9fdc6"
```

Comment:
This looks like something from some other pull request.

Reply:
This is a test stuff.